packages/getEntropy/snap.manifest.json

```diff
@@ -0,0 +1,27 @@
+{
+  "version": "5.2.0",
+  "description": "An example Snap that requests entropy.",
+  "proposedName": "@metamask/test-snap-getentropy",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MetaMask/test-snaps.git"
+  },
+  "source": {
+    "shasum": "dCzryw4PAs2AVT3fcdc0213RL3/+ZqSieeNTUua0sk8=",
+    "location": {
+      "npm": {
+        "filePath": "dist/bundle.js",
+        "packageName": "@metamask/test-snap-getentropy",
+        "registry": "https://registry.npmjs.org"
+      }
+    }
+  },
+  "initialPermissions": {
+    "endowment:rpc": {
+      "dapps": true
+    },
+    "snap_dialog": {},
```

Comment:
Is this permission needed?

Reply:
I was receiving an error that endowment:rpc, endowment:cronjob or one other needed to be there, so i used the same setup as bip-44 did. I haven't tried it after setting to false.

Reply:
I set it to false and it still works.

Reply:
Sorry I meant the `snap_dialog` permission. You need the `endowment:rpc` one in order to interact with it from the test snaps page.

Reply:
Oh i fixed that now too. Sorry. Got it.